source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -332,7 +332,25 @@ export class CardDesignerSurface {
 
         if (this.isPreviewMode) {
             cardToRender.onExecuteAction = (action: Adaptive.Action) => {
-                alert("Action executed\n" + JSON.stringify(action.toJSON(this._serializationContext), undefined, 4));
+                const actionType = action.getJsonTypeName();
+                let message = `Action executed: "${actionType}" title: "${action.title}"`;
+
+                const verb = (<Adaptive.ExecuteAction>action).verb;
```

Comment:
What does angular bracket do in `(<Adaptive.ExecuteAction>action).verb`?

Reply:
this is a TypeScript "[Type assertion](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions)", which can almost be thought of as a type cast :)